### PR TITLE
Prep for HTML-CSS material

### LINKED ISSUE #15: forgot grunt serve
Forgot to add command to run grunt serve

### LINKED ISSUE #16: Consider adding forms
Is there a way we can show forms here along with other html elements to show that html is larger than what we show them

### PR DESCRIPTION
**Pull request by berziiii**
- Update README.md
- clean up Barry's directory
- Linter error corrections
- Correct grammar

close #16 
close #15

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -133,6 +133,23 @@ typically in the the `<head>`. The syntax for using a `<link>` tag is `<link
 rel="stylesheet" type="text/css" href="...">` where 'href' is set to the
 location of the desired stylesheet.
 
+### Code-Along: Barry's Butcher Shop
+
+Together we are going to look at our good friend Barry's purposal for his
```
Comment (payne-chris-r) [inline README.md:144]: purposal

File: README.md [hunk 2]
```diff
@@ -133,6 +133,23 @@ typically in the the `<head>`. The syntax for using a `<link>` tag is `<link
...
+website. Barry has been so gracious to provide us with what he wants to be
+included on his website. Since Barry has spent his entire life practicing the
+ancient art of 'cutting up meat', he is not a technology oriented guy. Let's
+help out of good friend Barry!
```
Comment (payne-chris-r) [inline README.md:148]: - "help out our"

File: README.md [hunk 3]
```diff
@@ -133,6 +133,23 @@ typically in the the `<head>`. The syntax for using a `<link>` tag is `<link
...
+
+First, we are going to great a branch to create Barry's website.
```
Comment (payne-chris-r) [inline README.md:150]: *create not "great"

**Comment by berziiii** [comment]
See bf0e15f @payne-chris-r

File: README.md (hunk 1 above)
Comment (payne-chris-r) [inline README.md:144]: this one too @berziiii 
We can talk about if there's a better way to do this...

Comment (berziiii) [inline README.md:144]: see cfba722

**Pull request merged**